awesome_cart/awc.py

```diff
@@ -4,6 +4,7 @@
 import traceback
 import frappe
 import datetime
+import json
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [redefinition of unused 'json' from line 3](https://app.codacy.com/app/DigiThinkIT/awesome_cart/pullRequest?prid=4553813)